Dimitris/split.py

```diff
@@ -0,0 +1,11 @@
+import os.path as path
+import pandas as pd
+from pandas import DataFrame
+from sklearn.model_selection import train_test_split
+
+train_data_path = path.join(path.pardir, 'train.csv')
+data = pd.read_csv(train_data_path)
+new_test: DataFrame
```

Comment:
I only added this to help intellisense and forgot to delete it, but this is such a throwaway script that I'm not even going to amend it.